content/learn/manipulating-files/node-folders.en.md

```diff
@@ -126,7 +126,7 @@ fs.rmdir(dir, { recursive: true }, err => {
 });
 ```
 
-> **NOTE:** In Node `v16.x` the option `recursive` is **deprecated** for `fs.rmdir` of callback API, instead use `fs.rm` to delete folders that have content in them:
+<Alert> <b>NOTE:</b> In Node <code>v16.x</code> the option <code>recursive</code> is <b>deprecated</b> for <code>fs.mdir</code> of callback API, instead use <code>fs.rm</code> to delete folders that have content in them</Alert>
```

Comment:
Let's update the example to use the new way then instead of adding an alert?

As Node.js v16 is already "Maintenance LTS", meaning, people should already be using this new way.

Reply:
I updated the example as such:
![image](https://user-images.githubusercontent.com/84540554/199273269-6cae4cbb-a922-4ade-a185-660c2a766971.png)